PlayFabClient.go

```diff
@@ -324,7 +324,7 @@ func GetTitleData(keys []string, titleId string, secretKey string, logger Logger
 	return titlelData, nil
 }
 
-func GetStoreItems(storeId string, titleId string, playfabId string, catalogVersion string, secretKey string, logger Logger) ([]interface{}, error) {
+func GetStoreItems(storeId string, titleId string, playfabId string, catalogVersion string, secretKey string, logger Logger) ([]interface{}, interface{}, error) {
```

Comment:
Why the second argument is `interface{}`? don't we know what type it is going to be?